docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+version: "3"
+services:
+  node: 
+    image: victorabz/the-spinners
+    build: .
+    ports: 
+      - 3000:3000
+    command: npm start
+    depends_on: 
+      - db
+    environment:
+      - DB_URL=postgres://${POSTGRES_USER}:${POSTGRES_PASSWORD}@db:5432/${POSTGRES_DB}
```

Comment:
Do not hardcode this value instead get it from environment variable.

`postgres://${POSTGRES_USER}:${POSTGRES_PASSWORD}@db:5432/${POSTGRES_DB}` can be set manually in the .env file